docs/index.md

```diff
@@ -1,58 +1,26 @@
 ---
 id: index
-title: Welcome to Webaverse
+title: Welcome
 hide_title: true
 ---
 
 <div style='margin-top: -60px'></div>
-<h1 align="center"><b>🌌 Welcome To A Universe Of Apps ⚙️</b></h1>
+<h1 align="center"><b>Welcome To Webaverse</b></h1>
 
-<img alt="Webaverse world demo" src="/img/demo.gif" width="40%">
+<br/>
 
-<div align="center">
-  <a href="https://webaverse.com" target="_blank">Site</a>
-  &mdash;
-  <a href="https://github.com/webaverse/" target="_blank">Github</a>
-  &mdash;
-  <a href="https://discord.gg/MQNUGgB" target="_blank">Discord</a>
-  &mdash;
-  <a href="https://twitter.com/webmixedreality" target="_blank">Twitter</a>
-</div>
+There's a huge problem, which is, The Metaverse is already here -- it's just locked away in silos. <a href="https://webaverse.com/" target="_blank" rel="noopener noreferrer">The Webaverse</a> solves this by plugging into existing standards and communities in order to bootstrap the metaverse.
```

Comment:
This could just say what these docs will help you do; I'm not sure if it's the place for selling philosophy.

Reply:
a26d304